galleries/examples/specialty_plots/ishikawa_diagram.py

```diff
@@ -0,0 +1,200 @@
+"""
+================
+Ishikawa Diagram
+================
+
+Ishikawa Diagrams, fishbone diagrams, herringbone diagrams, or cause-and-effect
+diagrams are used to identify problems in a system by showing how causes and
+effects are linked.
+Source: https://en.wikipedia.org/wiki/Ishikawa_diagram
+
+"""
+import matplotlib.pyplot as plt
+
+from matplotlib.patches import Polygon, Wedge
+
+# Create the fishbone diagram
+fig, ax = plt.subplots(figsize=(10, 6), layout='constrained')
+ax.set_xlim(-5, 5)
+ax.set_ylim(-5, 5)
+ax.axis('off')
+COLOR = 'C0'
```

Comment:
You can use "tab:blue" to get the same color by a fixed name.

Reply:
I like keeping the default colors in the examples. This makes for a homogeneous visual impression and "brand identity". If there is a dedicated color configuration, its correct usage is implied. We don't have to choose another color just for the purpose of proving that this is properly applied.

Reply:
To be honest, I only made that variable to make it easier to experiment with different colours. I can just as easily remove it since it's only being used in 3 places.